eds4jinja2/builders/report_builder.py

```diff
@@ -97,4 +97,4 @@ def make_document(self):
         template.stream().dump(str(pathlib.Path(self.__OUTPUT_FOLDER__) / self.template))
 
         for listener in self.__after_rendering_listeners:
-            listener(str(self.__STATIC_FOLDER__), str(pathlib.Path(self.__OUTPUT_FOLDER__)))
+            listener(str(self.__STATIC_FOLDER__), str(pathlib.Path(self.__OUTPUT_FOLDER__)), self.configuration_context)
```

Comment:
since these parameters are useful in the __after_rendering_listeners, should they be passed to __before_rendering_listeners? Let's try to establish a fixed set of parameters that are available to the listeners:
option 1: decide a fixed set of parameters and use those, then this has to be described in the class documentation
option 2: pass self as the ultimate context carrier; still to be documented, with sample usages

Reply:
I would opt to pass self, but my years of .NET and Java makes me be affraid that someone would inadvertently change the variables in that "self" :)

Reply:
Let's go for option 1 then, where we pass the path to 
1. Either the "project folder" (the one containing the config.json)
1. or the path to the config.json
2. AND the path to the output folder
3. AND anything else needed?